src/fastoad/openmdao/variables/variable_list.py

```diff
@@ -121,16 +123,27 @@ def update(self, other_var_list: list, add_variables: bool = True):
             - if not, Variable instance from other_var_list will be added only if
               add_variables==True
 
+        The merge_metadata parameter controls how metadata is handled when replacing variables:
+            - if merge_metadata=False (default): The variable is completely overwritten with the
+              new one, and all previous metadata is lost (except for descriptions as noted above)
+            - if merge_metadata=True: Only the metadata explicitly provided in the new variable
```

Comment:
From how I understand the function `update_missing_metadata`, this should be changed to:
```suggestion
            - if merge_metadata=True: Only the metadata explicitly provided in the new variable and not present in the original variable
```
Although I don't know if that was the intended behaviour.

Reply:
Well actually I think the description is misleading. I understood it the other-way-around.

Should it be:
`if merge_metadata = True : The metadata present in the original variable and absent from the new variable will be conserved. All other metadata will be overwritten by the ones in new variable.`

Reply:
Yes is much clearer like this!